common/src/main/java/com/wynntils/wynn/model/map/MapModel.java

```diff
@@ -36,6 +52,60 @@ public static List<MapProfile> getMaps() {
         return maps;
     }
 
+    public static Set<Poi> getAllPois() {
+        return allPois;
+    }
+
+    public static void loadLabels() {
+        File mapDirectory = new File(WebManager.API_CACHE_ROOT, "maps");
+
+        String url = "https://raw.githubusercontent.com/Wynntils/Reference/main/locations/places.json";
```

Comment:
Perhaps this could be extracted to the top of the class. Perhaps, it is fine for it to stay here.

Reply:
Yeah, no, that is crap. :-) Just wanted to get this starting, and I was kind of half sleeping in front of the computer yesterday night.